<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>excel_lite</class>
 <widget class="QMainWindow" name="excel_lite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>excel_lite</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>801</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="find">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MIN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MAX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ARITH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SUM</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="sort">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>QSORT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MONKEY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COMB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BUBBLE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GNOME</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="fill">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ZERO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RANDOM</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sort_b">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_b">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>FIND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_b">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>FILL</string>
    </property>
   </widget>
   <widget class="QLabel" name="find_output">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="size_r">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="size_c">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="size_b">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="size_r_add">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>25</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="size_r_del">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>60</y>
      <width>25</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="size_c_add">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>25</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="size_c_del">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>60</y>
      <width>25</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_duplicates">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>DEL DBLCT</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="search">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search_b">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="search_output">
    <property name="geometry">
     <rect>
      <x>799</x>
      <y>90</y>
      <width>201</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sort_status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>unsorted</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="search_set">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LINE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BINARY</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="remove_duplicates_error">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
